FullDemo/playkitdemo/src/main/java/com/kaltura/playkitdemo/LoadControlSetup.java

```diff
@@ -0,0 +1,121 @@
+package com.kaltura.playkitdemo;
+
+import android.os.AsyncTask;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.google.gson.JsonPrimitive;
+import com.kaltura.netkit.connect.executor.RequestQueue;
+import com.kaltura.playkit.Player;
+import com.kaltura.playkit.player.LoadControlBuffers;
+
+import java.io.IOException;
+
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+class LoadControlSetup {
+
+    public static final String GET_UICONF_DATA = "https://cdnapisec.kaltura.com/api_v3/?format=1&partnerId=1982551&service=uiconf&action=get&id=43565151";
+
+    private static Integer experimentId;
+
+    private static Integer initialBuffer;
+    private static Integer afterRebuffer;
+    private static Integer bufferLength;
+    //private static int startBitrate;
+
+    private static boolean loaded = false;
+
+    // Call once as soon as the app loads, in Application.onCreate()
+    public static void load() {
+        if (loaded) {
+            return;
+        }
+
+        AsyncTask.execute(() -> {
+            synchronized (LoadControlSetup.class) {
+                if (!loaded) {
+                    OkHttpClient client = new OkHttpClient();
+                    Request request = new Request.Builder().url(GET_UICONF_DATA).build();
+                    try {
+                        Response response = client.newCall(request).execute();
```

Comment:
It's probably better to use `client.newCall(request).enqueue()` instead of execute -- it executes the request in the background and calls a callback. Then we can get rid of the AsyncTask.